TicketsDemo.Xml/XMLWriter.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml.Serialization;
+using TicketsDemo.Data.Entities;
+using TicketsDemo.Domain.Interfaces;
+using TicketsDemo.EF;
+using TicketsDemo.Xml.Interfaces;
+
+namespace TicketsDemo.Xml
+{
+    public class XMLWriter : IXMLWriter
+    {
+        private ISettingsService _settingsService;
+
+        public XMLWriter(ISettingsService settingsService)
+        {
+            _settingsService = settingsService;
+        }
+
+        public void WriteXML()
```

Comment:
XMLWriter name makes an impression that this is generic service, responsible for writing some obstract XML. In fact it is not, it is just seeding the initial data, taking it from Entity Framework. Please rename the class (and method) to reflect it's actual purpose